src/components/NonProfit/NonProfit.js

```diff
@@ -111,6 +115,17 @@ export default function NonProfit(props) {
     } else {
       if (nonprofit?.problem_statements?.length === 0) {
         return <div>Working on it!</div>;
+      } else if (!nonprofit?.problem_statements) {
```

Comment:
Likely not what you want @CobyW98 - using ? within an else if/if statement is a bit confusing/hard to understand since that is doing the null handling for you. 

In this case, the length == 0 will likely branch there and nothing will run this code unless the problem statements are null (and not an empty list `[]`).

Here, we'd likely want to remove the "Working on it" and just put the code you've added